modules/openapi-generator/src/main/resources/kotlin-client/jvm-common/infrastructure/Serializer.kt.mustache

```diff
@@ -22,12 +22,10 @@ import org.threeten.bp.OffsetDateTime
 import java.util.UUID
 {{/gson}}
 {{#jackson}}
-import com.fasterxml.jackson.databind.ObjectMapper
-import com.fasterxml.jackson.databind.SerializationFeature
-import com.fasterxml.jackson.datatype.jdk8.Jdk8Module
-import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule
-import com.fasterxml.jackson.annotation.JsonInclude
-import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
+    import com.fasterxml.jackson.databind.ObjectMapper
+    import com.fasterxml.jackson.databind.SerializationFeature
+    import com.fasterxml.jackson.annotation.JsonInclude
+    import com.fasterxml.jackson.module.kotlin.jacksonObjectMapper
```

Comment:
@zapodot thanks for the PR. What about removing these additional spaces?

Reply:
Done!